cmd/testdata/create/one/a/b/bar/baz.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package bat;
+
+option go_package = "batpb";
+option java_multiple_files = true;
+option java_outer_classname = "BazProto";
+option java_package = "com.bat";
```

Comment:
missing new lines at the end of these proto files

Reply:
Second thought, why are we introducing these files?

Reply:
Oof, I forgot to update the `.gitignore` in my project-wide find/replace. Fixed 👍